example/my-app/src/components/tables/ExecutionsTableSimple.tsx

```
@@ -0,0 +1,637 @@
+const STATE_TYPE_CONFIG: Record<string, { label: string; color: string; urlParam: string }> = {
+  'workflow.Done': { label: 'Done', color: '#10b981', urlParam: 'done' },
+  'workflow.Error': { label: 'Error', color: '#ef4444', urlParam: 'error' },
+  'workflow.Await': { label: 'Await', color: '#3b82f6', urlParam: 'await' },
+  'workflow.Scheduled': { label: 'Scheduled', color: '#eab308', urlParam: 'scheduled' },
+  'workflow.ScheduleStopped': { label: 'Paused', color: '#6b7280', urlParam: 'paused' },
+  'workflow.NextOperation': { label: 'Next', color: '#a855f7', urlParam: 'next' },
+}
+
+// Reverse mapping for URL params to state types
+const URL_TO_STATE_TYPE: Record<string, string> = Object.entries(STATE_TYPE_CONFIG).reduce(
+  (acc, [stateType, config]) => ({ ...acc, [config.urlParam]: stateType }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)